### PR TITLE
Fixes #6832

### LINKED ISSUE #6832: Line number missing in stdlib trace
```nim
import strutils
echo "".split("")
```

```
a1.nim(2)                a1
strutils.nim             split
system.nim(3740)         raiseAssert
system.nim(2809)         sysFatal
Error: unhandled exception:
0 < len(sep)  [AssertionError]
```

### PR DESCRIPTION
**Pull request by data-man**
Maybe there is a better way.

### CODE REVIEW

File: lib/system.nim [hunk 1]
```diff
@@ -3805,10 +3805,10 @@ template assert*(cond: bool, msg = "") =
 template doAssert*(cond: bool, msg = "") =
   ## same as `assert` but is always turned on and not affected by the
   ## ``--assertions`` command line switch.
-  bind instantiationInfo
```
Comment (Araq) [inline lib/system.nim]: Why remove this bind?

**Comment by data-man** [comment]
Now for the code from the issue:
```nim
import strutils
echo "".split("")
```
outputs:
> i_6832.nim(2)            i_6832
strutils.nim             split
system.nim(3839)         raiseAssert
system.nim(2849)         sysFatal
Error: unhandled exception: 
0 < len(sep) lib/pure/strutils.nim(689)  [AssertionError]

File: lib/system.nim (hunk 1 above)
Comment (Araq) [reply lib/system.nim]: Keep it.

Comment (data-man) [reply lib/system.nim]: Done.

**Pull request merged**